### PR TITLE
Add WecOptTool

### PR DESCRIPTION
**Pull request by michaelcdevin**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by michaelcdevin** [comment]
@conda-forge/help-python, ready for review!

**Comment by michaelcdevin** [comment]
@cmichelenstrofer, can you confirm here that you approve of being a maintainer for this package?

**Comment by cmichelenstrofer** [comment]
> @cmichelenstrofer, can you confirm here that you approve of being a maintainer for this package?

I approve of being a maintainer for this package.

File: recipes/wecopttool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+about:
+  home: https://github.com/SNL-WaterPower/WecOptTool
+  summary: WEC Design Optimization Toolbox
+  description: |
+    The Wave Energy Converter Design Optimization Toolbox (WecOptTool)
+    allows users to perform wave energy converter (WEC) device design
+    optimization studies with constrained optimal control.
+  license: GPL-3.0-only
+  license_family: GPL
```
[the remark this answers is not in the document]
Comment (michaelcdevin) [reply recipes/wecopttool/meta.yaml]: Thanks! This has been corrected.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**